.infrastructure/hpa.yml

```diff
@@ -2,7 +2,7 @@ apiVersion: autoscaling/v2
 kind: HorizontalPodAutoscaler
 metadata:
   name: todoapp
-  namespace: todoapp
+  namespace: mateapp
```

Comment:
The namespace 'mateapp' is specified here. Ensure that this namespace exists in your Kubernetes cluster before applying this configuration. If it doesn't exist, you may need to create it or change this value to a namespace that does exist.